client/debug/main.go

```diff
@@ -63,10 +64,16 @@ func getCodecInterfaces() *cobra.Command {
 	return &cobra.Command{
 		Use:   "list-interfaces",
 		Short: "List all registered interface type URLs",
-		Long:  "List all registered interface type URLs using the application codec",
+		Long: fmt.Sprintf(`List all registered interface type URLs using the application codec.
+
+Example:
```

Comment:
Can you use the Example field instead?

Reply:
Oh! My bad, I mindlessly copy/pasta'd what I saw below for the pub key cmd.
I'll make the change, if you want I can also edit the pub key examples to use the example field also!

Reply:
Sure!